### PR TITLE
fix for "send max token is not working"

### PR DESCRIPTION
**Pull request by maycon-mello**
https://dock-team.atlassian.net/browse/DCKA-887

### CODE REVIEW

File: src/features/transactions/transactions-slice.js [hunk 1]
```diff
@@ -183,6 +190,7 @@ export const transactionsOperations = {
         .sendTokens({
           toAddress: recipientAddress,
           fromAddress: accountAddress,
+          transferAll: sendMax,
```
Comment (mike-parkhill) [inline src/features/transactions/transactions-slice.js:193]: what's this used for?

Comment (mike-parkhill) [reply src/features/transactions/transactions-slice.js:204]: do we need unit test updates to reflect this change?

Comment (maycon-mello) [reply src/features/transactions/transactions-slice.js:204]: there is no test for this yet, worth adding integration tests

**Comment by maycon-mello** [comment]
> side comment but noticed in this PR. why are we committing the bundle JS? shouldnt it be ignored in favour of building it from source?

I agree we should be able to add it as part of ci, let me create a jira for it

**Comment by maycon-mello** [comment]
There is a command already, `yarn build-sdk` it will generate the bundle JS, should be a quick fix actually

**Comment by maycon-mello** [comment]
> There is a command already, `yarn build-sdk` it will generate the bundle JS, should be a quick fix actually

since we have merged the fix already I did a rebase on this one, so that we are not pushing the bundle anymore

**Review by SamHellawell — APPROVED** [review]

**Pull request merged**